cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -1384,7 +1385,7 @@ def _add_outputs(self):
         )
 
 
-class ComputeFleetConstruct(Construct):
+class ComputeFleetStack(NestedStack):
     """Construct defining compute fleet specific resources."""
```

Comment:
nit - change doc string

Reply:
As discussed, this will likely be a construct in a subsequent PR.